fontbakery.js

```diff
@@ -0,0 +1,316 @@
+
+const SORT_RESULT = {
+  'FAIL': 'aa',
+  'WARN': 'bb',
+  'INFO': 'cc',
+  'ERROR': 'dd',
+  'PASS': 'ee',
+  'SKIP': 'zz',
+};
+
+const NOWASM = (s) => `
+This check cannot be run in the web environment. This is because ${s}.
+The web version of fontbakery is not a full replacement for the Python
+version, and we recommend that you install fontbakery and check your
+fonts locally to ensure that all checks are run.`;
+const CANT_COMPILE = (s) => NOWASM(`the ${s} library cannot be compiled for WASM`);
+const NEEDS_NETWORK = NOWASM("it needs access to the network");
+const EXCUSES = {
+  // Needs dependencies
+  'com.adobe.fonts/check/freetype_rasterizer': CANT_COMPILE('Freetype'),
+  'com.google.fonts/check/ots': CANT_COMPILE('OpenType Sanitizer'),
+  'com.google.fonts/check/alt_caron:googlefonts': NOWASM('the check requires a library (babelfont) with a Rust dependency'),
+  // Needs network
+  'com.google.fonts/check/vendor_id': NEEDS_NETWORK,
+  'com.google.fonts/check/fontdata_namecheck': NEEDS_NETWORK,
+  'com.google.fonts/check/vertical_metrics_regressions': NEEDS_NETWORK,
+  'com.google.fonts/check/metadata/includes_production_subsets': NEEDS_NETWORK,
+  'com.google.fonts/check/metadata/designer_profiles': NEEDS_NETWORK,
+  'com.google.fonts/check/description/broken_links': NEEDS_NETWORK,
+  'com.google.fonts/check/metadata/broken_links': NEEDS_NETWORK,
+  'com.google.fonts/check/version_bump': NEEDS_NETWORK,
+  'com.google.fonts/check/production_glyphs_similarity': NEEDS_NETWORK,
+  // Shaping checks
+  'com.google.fonts/check/render_own_name': NOWASM('it requires the Freetype library, which is unavailable in WASM'),
```

Comment:
This is not clear to me. Why isn't this simply `CANT_COMPILE('Freetype')`? Is there any semantic difference, in comparison to the situation of the other libs mentioned here?